packages/unlock-js/src/PublicLock/v4/index.js

```diff
@@ -12,6 +13,7 @@ export default {
   purchaseKey,
   purchaseKeys,
   grantKey,
```

Comment:
Let's make sure to refactor `grantKey` so it uses `grantKeys` to make things DRY :)

Reply:
done